vividus-extension-selenium/src/main/java/org/vividus/ui/action/search/SearchAttributes.java

```diff
@@ -32,15 +33,12 @@
 
     public SearchAttributes(IActionAttributeType searchAttributeType, SearchParameters searchParameters)
     {
-        if (searchAttributeType.getSearchLocatorBuilder() == null)
-        {
-            checkIfApplicable(searchAttributeType, IElementSearchAction.class, "Search");
-        }
+        checkIfApplicable(searchAttributeType, IElementSearchAction.class, "Search");
         this.searchAttributeType = searchAttributeType;
         this.searchParameters = searchParameters;
     }
 
-    public SearchAttributes(ActionAttributeType searchAttributeType, String searchValue)
+    public SearchAttributes(IActionAttributeType searchAttributeType, String searchValue)
```

Comment:
Since you're renaming ActionAttributeType can we use some meaningful name? SearchType or something else

Reply:
renamed to ElementActionType